packages/next/client/image.tsx

```diff
@@ -87,8 +87,7 @@ function getDeviceSizes(width: number | undefined): number[] {
   if (typeof width !== 'number') {
     return configDeviceSizes
   }
-  const smallest = configDeviceSizes[0]
-  if (width < smallest && configIconSizes.includes(width)) {
```

Comment:
What happened to this logic?

Reply:
We found a use case where we would want to opt-out of the srcset behavior, even with a slightly larger image (for example 400px)

Reply:
A better solution might be a another property on the component but then you have two things you need to configure because we still need to know the device sizes